core/domain/question_domain.py

```diff
@@ -213,6 +213,34 @@ def _convert_state_v29_dict_to_v30_dict(cls, question_state_dict):
 
         return question_state_dict
 
+    @classmethod
+    def _convert_state_v30_dict_to_v31_dict(cls, question_state_dict):
+        """Converts from version 30 to 31. Version 31 updates the
+        Voiceover model to have an initialized duration attribute of 0.0.
+        This will be updated when a new mp3 audio file is uploaded
```

Comment:
Can you explain this line? is it for the exiting audio or the new audio?

Reply:
It is supposed to be existing. I'll clear that up.

Reply:
Done.